packages/vuetify/src/mixins/menuable.ts

```diff
@@ -34,23 +56,28 @@ const dimensions = {
  * As well as be manually positioned
  */
 /* @vue/component */
-export default Vue.extend({
+export default mixins<options &
+/* eslint-disable indent */
+  ExtractVue<[
+    typeof Stackable,
+    typeof Positionable
+  ]>
+/* eslint-enable indent */
+>(
+  Stackable,
+  Positionable
+  /* @vue/component */
+).extend({
   name: 'menuable',
 
-  mixins: [
-    Positionable,
-    Stackable
-  ],
-
   props: {
     activator: {
       default: null,
-      validator: val => {
+      validator: (val: string | object) => {
         return ['string', 'object'].includes(typeof val)
       }
-    },
+    } as PropValidator<string | HTMLElement>,
```

Comment:
that should be `string|HTMLElement|VNode` i guess

Reply:
What would make this a VNode?